ruby/grape/config.ru

```diff
@@ -0,0 +1,25 @@
+Bundler.require :default
+
+module Bench
+  class BaseAPI < Grape::API
+    get '' do
+      ''
+    end
+  end
+
+  class UserAPI < Grape::API
+    get '/user/:id' do
+      params[:id]
+    end
+    post '/user' do
+      ''
```

Comment:
A big part of performance benchmarking is serializing responses. I suggest returning JSON here.

Reply:
it will be done later, see https://github.com/the-benchmarker/web-frameworks/issues/1728 but not now (this requires much much work)